lib/rspec/mocks/error_generator.rb

```diff
@@ -65,7 +65,7 @@ def raise_missing_default_stub_error(expectation, *args)
         diff = diff_message(expectation.expected_args, args)
 
         message = default_error_message(expectation, expected_args, actual_args)
-        message << "Diff:\n #{diff}" unless diff.empty?
+        message << "\n Diff:\n #{diff}" unless diff.empty?
```

Comment:
In `raise_unexpected_message_args_error`, we use `"\nDiff"`, which left-aligns `Diff` against the margin, whereas this has Diff indented one space.  I don't think we want that space.